lighty-examples/lighty-gnmi-community-restconf-app/README.md

```diff
@@ -0,0 +1,81 @@
+# lighty.io gNMI/RESTCONF application
+This application provides gNMI south-bound utilized with RESTCONF interface to manage gNMI devices on the network.
+Application works as standalone SDN controller. It is capable to connect to gNMI devices and expose connected devices
+over RESTCONF north-bound APIs. In this application will be started gNMI simulator as gNMI target and all operation will
+be performed on this device.
+
+Application lighty.io gNMI/RESTCONF is pre-prepared with [Openconfig YANG models](src/main/assembly/resources/yangs).
+These models are used by both gNMI application and gNMI device simulator. Device have already prepared state/config data
+configured by [this](src/main/assembly/resources/simulator/initialJsonData.json) json file.
+For communication with gNMI device is required to use TLS communication with certificates and authorized
+by username and password.
+
+This application starts:
+* [Lighty.io Controller](https://github.com/PANTHEONtech/lighty/tree/master/lighty-core/lighty-controller)
+* [lighty.io RESTCONF](https://github.com/PANTHEONtech/lighty/tree/master/lighty-modules/lighty-restconf-nb-community)
+* [lighty.io gNMI](https://github.com/PANTHEONtech/lighty/tree/master/lighty-modules/lighty-gnmi/lighty-gnmi-sb)
+* [lighty.io gNMI device simulator](https://github.com/PANTHEONtech/lighty/tree/14.0.x/lighty-modules/lighty-gnmi/lighty-gnmi-device-simulator)
+
+## Prerequisites
+In order to build and start and use the lighty.io gNMI/RESTCONF application locally, you need:
+* Java 11 or later
+* Maven 3.5.4 or later
+* Postman v7.36.5. or later
+
+## Build and start
+To build and start the lighty.io gNMI/RESTCONF application in your local environment, follow these steps:
+1. Build the application using maven
+   `mvn clean install`
+
+2. Unpack the application ZIP distribution created in the _lighty-gnmi-community-restconf-app/target_ called
+   `lighty-gnmi-community-restconf-app-<version>-bin.zip`
+
+3. Start the application by:
+   - Running it's _.jar_ file `java -jar lighty-gnmi-community-restconf-app-<version>-SNAPSHOT.jar`
+   - Or with provided script `./start-controller.sh`
+
+## Example of using lighty.io gNMI/RESTCONF application
+This example show how to connect gNMI device and perform basic CRUD operation on the device. All RESTCONF request
+used in this example are provided in [postman-collection](lighty.io gNMI-RESTCONF application.postman_collection.json).
+
+ - ### Add client certificates to lighty.io gNMI keystore
+Used certificates can be found [here](src/main/assembly/resources/certificates). To keystore is added only client
```

Comment:
only client certificates are added to keystore

Reply:
done